src/main/charts/confluence/templates/_helpers.tpl

```diff
@@ -176,6 +176,19 @@ The command that should be run by the nfs-fixer init container to correct the pe
 {{- end }}
 {{- end }}
 
+{{ define "confluence.volumeMounts" }}
```

Comment:
Could we add some documentation why we are mounting into two places under the same name (application and access logs availability)?

Reply:
Done